dags/looker.py

```diff
@@ -114,7 +114,7 @@
         ],
         task_id="lookml_generator",
         name="lookml-generator-1",
-        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:" + image_tag,
+        image="us-docker.pkg.dev/moz-fx-data-artifacts-prod/firefox-public-data-report-etl/firefox-public-data-report-etl:" + image_tag,
```

Comment:
Copied the wrong thing
```suggestion
        image="us-docker.pkg.dev/moz-fx-data-artifacts-prod/lookml-generator/lookml-generator:" + image_tag,
```

Reply:
right 🤦‍♀️